compact.cpp

```diff
@@ -32,7 +32,7 @@ unsigned __stdcall ObjectMemory::lastOTEntry()
 	const OTE* pOT = m_pOT;
 	while (pOT[i].isFree())
 	{
-		ASSERT(i >= 0);
+		ASSERT(0 < i);
```

Comment:
@jgfoster it's not a big deal, so your choice. Looks like it needs a manual merge now anyway. Thanks.